Assets/Scripts/GrappleGunBehavior.cs

```diff
@@ -47,12 +49,31 @@ private bool GrappleTargetDestroyed()
         return grappling && !grappledObj;
     }
 
+    private void BuildGrapple(RaycastHit hit)
+    {
+        joint = player.AddComponent<SpringJoint>();
+        joint.autoConfigureConnectedAnchor = false;
+        joint.anchor = Vector3.zero;
+        joint.connectedBody = hit.rigidbody;
+        if (grappledObjOffset.HasValue)
+        {
+            joint.connectedAnchor = grappledObjOffset.Value.normalized;
```

Comment:
why are you normalizing the offset?

Reply:
For some weird reason, joints work in a strange way, where the connectedAnchor is proportional local coords where 0.5 is 0.5 of the object's size away from the center, so I normalize it to place the offset in the right spot.

Reply:
I searched for what you were talking about but all I found is that the coordinates are in local space not world space. That doesn't mean 0.5 is 0.5 of the object's size away from the center. What I found says you should use `grappledTransform.InverseTransformPoint(hit.point)`. But if you don't want to do that I can do it when I merge in your changes.

Reply:
I'm leaving this up to you